README.md

```diff
@@ -1084,6 +1085,27 @@ module.exports = {
 };
 ```
 
+## `compareThreshold`
```

Comment:
I think this section should come with a warning that helps people understand that using this can unintentionally hide diffs that they don't want hidden, so it needs to be used carefully.

Also, it might make sense to pick a different name that indicates this a little more clearly. Maybe something that says `autoIgnoreDiffsUnderThreshold` or something like that?

Reply:
I added a warning. 

About the name, I wanted this to have the word "compare" in it so that it's clearly connected to the `happo compare` call. I thought about putting it inside a `compare` key, so something like 

```js
{ compare: { threshold: 0.05 }}
```
but in the end I decided to keep it simple. I'm going to err on the side of memorability here. There are more specfic names but I can't come up with one that feels good.